src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallMapper;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    @Autowired
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @GetMapping("/") // /cinema-halls/
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
+        return cinemaHallService.getAll().stream()
+                .map(cinemaHallMapper::modelToDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping // /cinema-halls{"capacity":200, "description":"cinema hall 1"}
```

Comment:
```suggestion
    @PostMapping
```

lets remove all comments